app/views/courses/_advice.html.erb

```diff
@@ -5,4 +5,4 @@
 <p class="govuk-body">Register with <%= link_to 'Get Into Teaching', 'https://beta-getintoteaching.education.gov.uk/helping-you-become-a-teacher', class: "govuk-link" %>, the Department for Education’s free support and advice service, for personalised guidance from teaching experts. They can help you to prepare your application, book school experience, and access exclusive teaching events. You can also call them free of charge on 0800 389 2500, or speak to an adviser using their <%= link_to "online chat service", "https://beta-getintoteaching.education.gov.uk/#talk-to-us", class: 'govuk-link' %>, from 8am to 8pm, Monday to Friday.</p>
 
 <h4 class="govuk-heading-m">Is there something wrong with this page?</h4>
-<p class="govuk-body">If there is something wrong with this course listing, <%= bat_contact_mail_to "contact us by email", subject: "Edit #{course.name} (#{course.provider_code}/#{course.course_code})" %>.</p>
+<p class="govuk-body">If there is something wrong with this course listing, <%= bat_contact_mail_to "contact us by email", subject: "Edit #{course.name} (#{course.provider_code}/#{course.code})" %>.</p>
```

Comment:
```suggestion
<p class="govuk-body">If there is something wrong with this course listing, <%= bat_contact_mail_to "contact us by email", subject: "Edit #{course.name} (#{course.provider.code}/#{course.code})" %>.</p>
```
Maybe?


Coming to think of it the model decorator or the model itself may be nicer to to remap 

`course.has_vacancies?` vs `course.has_vacancies`
`course.course_code` vs `course.code`
`course..provider_code` vs `course.provider.code`
`provider.provider_code` vs `provider.code`
`provider.provider_name` vs `provider.name`

Reply:
I've fixed the code for now. Will refactor the decorator in another PR